src/runner/globals.js

```diff
@@ -23,7 +25,7 @@ const SHARED_MIXED_OPS = {
 
 const isDecInstance = (a) => a instanceof Decimal128 || a instanceof Big;
 
-const binaryExpressionHandler = (left, right, op, message) => {
+globalThis.binaryExpressionHandler = (left, right, op, message) => {
```

Comment:
Actually, having reviewed the code for blog posts, I think I am okay keeping this here, since we reach out in a few other places, we may as well be consistent. At least the file is clear that this should be limited.

Reply:
So, the issue is that this file was already adding all these things to the global object, because this file gets evaluated in the global page context: https://github.com/sarahghp/decimal-playground/blob/main/src/runner/index.html#L8. If you load the playground without this change and input into the code pane e.g.
```js
console.log(binaryExpressionHandler);
console.log(SHARED_SINGLE_OPS);
```
you get the values that are defined in this global.js file, regardless of this change. So I think it's actually better if it's explicit that all these things are getting put onto `globalThis`, maybe even the ones that the linter doesn't complain about because they are used within this file.

Although there are a few which actually don't need to be exposed; `unaryEvaluators` is only used within `wrappedUnaryHandler`, and `SHARED_SINGLE_OPS` and `SHARED_MIXED_OPS` are duplicated in constants.js, from where they could be imported instead; we could do these inside an IIFE or something, or even an explicit `defineGlobals(globalThis)` step, so that everything put on the global object is done deliberately. I'll implement this in the linter PR but in a separate commit so we can drop it if you don't prefer that approach.

Reply:
→ #49